presto/Dockerfile

```
@@ -4,13 +4,21 @@ LABEL maintainer="https://www.starburstdata.com/"
 ARG presto_version=0.195-e.0.6
 ARG dist_location=https://s3.us-east-2.amazonaws.com/starburstdata/presto/starburst/195e/${presto_version}
 
+COPY ./installdir /installdir
+
 RUN set -xeu && \
     yum install -y util-linux `# uuidgen; TODO remove after https://github.com/prestodb/presto/pull/10670` && \
     yum -y localinstall "${dist_location}/presto-server-rpm-${presto_version}.x86_64.rpm" && \
-    curl -fsSL "${dist_location}/presto-cli-${presto_version}-executable.jar" -o /usr/local/bin/presto-cli && \
+    cli_location="${dist_location}/presto-cli-${presto_version}-executable.jar" && \
```

I find it a bit awkward that `installdir` is hardcoded in this script and user has to pass matching `--build-arg dist_location=/installdir`.

What if you do something like:
```
cli_location="${dist_location}/presto-cli-${presto_version}-executable.jar"
if test -f "/installdir/presto-cli-${presto_version}-executable.jar"; then
     cli_location="/installdir/presto-cli-${presto_version}-executable.jar"
fi
```
and the same for rpm location.

Thanks to above there would be no need for `--build-arg dist_location=/installdir`.

In case when binary files versions do not match `ARG presto_version`, then it would be nice to raise an exception instead of fallback to using default location.

What do you think?